apps/astarte_appengine_api/lib/astarte_appengine_api_web/plug/guardian_authorize_path.ex

```diff
@@ -22,7 +22,7 @@ defmodule Astarte.AppEngine.APIWeb.Plug.GuardianAuthorizePath do
   defp authorize(conn, opts) do
     with %User{authorizations: authorizations} <- AuthGuardian.Plug.current_resource(conn),
          {:ok, auth_path} <- build_auth_path(conn),
-         :ok <- is_path_authorized?(conn.method, auth_path, authorizations) do
+         :ok <- path_authorized?(conn.method, auth_path, authorizations) do
```

Comment:
as we're reworking naming anyway, it's a good time to make sure they're not only accepted by credo, but that they also make sense

in this instance, a function named with a question mark at the end is expected to return a boolean, while this function really returns a result tuple

I suggest changing its name to better reflect this behavior. Usually, we just add `check_` at the start


```suggestion
         :ok <- check_path_authorized(conn.method, auth_path, authorizations) do
```